RedHat-UX/red-hat-design-system
# docs: update accordion header markup after charmander release

## What I did

1. Now that Charmander is out, heading tags go on the outside of `<rh-accordion-header>`(instead of inside).
2. This PR fixes that nesting for all of our docs on uxdot.

## Testing Instructions

1. Go to the DP
2. View the Elements > [Accordion overview page](https://deploy-preview-1805--red-hat-design-system.netlify.app/elements/accordion/)
3. View other pages with accordions like [Disclosure](https://deploy-preview-1805--red-hat-design-system.netlify.app/patterns/disclosure/) and [Designers](https://deploy-preview-1805--red-hat-design-system.netlify.app/get-started/designers/#frequently-asked-questions)
4. Verify the markup is correct

## Notes to Reviewers

Looks like [these changes](https://github.com/RedHat-UX/red-hat-design-system/pull/1715) in Charmander added a regression where the accordion header text is not bold. You will see this in this PR. I am going to open a separate PR with a changeset to make sure accordion header text gets bolded.
